checks/check_extra7160

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2019) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+CHECK_ID_extra7160="7.160"
+CHECK_TITLE_extra7160="[extra7160] Check if elastic IP addresses with associations are protected by AWS Shield"
+CHECK_SCORED_extra7160="NOT_SCORED"
+CHECK_CIS_LEVEL_extra7160="EXTRA"
+CHECK_SEVERITY_check="Medium"
+CHECK_ASFF_RESOURCE_TYPE_extra7160="AwsEc2Eip"
+CHECK_ALTERNATE_extra7160="extra7160"
+CHECK_SERVICENAME_extra7160="ec2"
+CHECK_REMEDIATION_extra7160='Add as a protected resource in AWS Shield'
+
+extra7160() {
+  if [[ "$(aws shield get-subscription-state --out text)" == "ACTIVE" ]]; then
+    CALLER_IDENTITY=$($AWSCLI sts get-caller-identity $PROFILE_OPT --query Arn)
+    PARTITION=$(echo $CALLER_IDENTITY | cut -d: -f2)
+    ACCOUNT_ID=$(echo $CALLER_IDENTITY | cut -d: -f5)
+    for regx in $REGIONS; do
```

Comment:
Isn't it a global service? In that case, we don't need the `REGIONS` for loop

Reply:
Elastic IP addresses are allocated within a region, so, to capture them all, you need to loop through all regions.